pkg/bigquery/routes/routes.go

```diff
@@ -59,6 +63,7 @@ func (r *ResourceHandler) tableSchema(rw http.ResponseWriter, req *http.Request)
 
 func (r *ResourceHandler) Routes() map[string]func(http.ResponseWriter, *http.Request) {
 	return map[string]func(http.ResponseWriter, *http.Request){
+		"/projects":             r.projects,
```

Comment:
Should be named default-project, no?

Reply:
Forgot to remove it at all :) This route ain't used anymore